modules/client/src/types.ts

```diff
@@ -2,75 +2,38 @@ import { IMessagingService } from "@connext/messaging";
 import {
   AppRegistry,
   AppState,
+  BigNumber as connextBN,
   ChannelProvider,
   ChannelState,
+  ClientOptions,
   GetConfigResponse,
   MultisigState,
+  Store,
 } from "@connext/types";
 import { Node as CFCoreTypes } from "@counterfactual/types";
-import { providers, utils, Wallet } from "ethers";
+import { providers, Wallet } from "ethers";
+import { Network } from "ethers/utils";
 
 import { ChannelRouter } from "./channelRouter";
 import { NodeApiClient } from "./node";
 
-export type BigNumber = utils.BigNumber;
-export const BigNumber = utils.BigNumber;
-
-export interface Store extends CFCoreTypes.IStoreService {
-  set(
-    pairs: {
-      path: string;
-      value: any;
-    }[],
-    shouldBackup?: Boolean,
-  ): Promise<void>;
-  restore(): Promise<{ path: string; value: any }[]>;
-}
-
-export interface ClientOptions {
-  // provider, passed through to CF node
-  ethProviderUrl: string;
-  // node information
-  nodeUrl: string; // ws:// or nats:// urls are supported
-
-  // signing options, include either a mnemonic directly
-  mnemonic?: string;
-
-  // or a channel provider
-  channelProvider?: ChannelProvider;
-  // function passed in by wallets to generate ephemeral keys
-  // used when signing applications
-  keyGen?: () => Promise<string>; // TODO: what will the type look like?
-  safeSignHook?: (state: ChannelState | AppState) => Promise<string>;
-  store: Store;
-  // TODO: state: string?
-  logLevel?: number; // see logger.ts for meaning, optional
-  // TODO: should be used in internal options? --> only if hardcoded
-  // nats communication config, client must provide
-  natsClusterId?: string;
-  natsToken?: string;
-}
+export type BigNumber = connextBN;
+export const BigNumber = connextBN;
 
 export type InternalClientOptions = ClientOptions & {
   appRegistry: AppRegistry;
   channelRouter: ChannelRouter;
+  channelProvider?: ChannelProvider;
   config: GetConfigResponse;
   contract?: MultisigState;
```

Comment:
hm, i know this isnt with this PR but not sure how i didn't see this earlier, what is this:
```typescript
type MultisigState<T = string> = {
    id: number;
    xpubA: string;
    xpubB: string;
    multisigAddress: string;
    freeBalanceA: T;
    freeBalanceB: T;
    appIds: number[];
}
```

the free balance would never be just a `string` or a `BigNumber` and im not sure how it's being used here or in instantiation?

also, why is the appId a `number[]`? the only IDs we ever use are strings

Is this something that gets pulled by the hub? can we delete?

Reply:
i deleted it